packages/babel-plugin-transform-arrow-functions/test/fixtures/arrow-functions/arguments-global-var/output.js

```diff
@@ -1,7 +1,7 @@
 var _arguments2 = 1;
 
 function fn() {
-  var _arguments = _arguments2;
+  var _arguments = arguments;
```

Comment:
Note for self and others: this is correct, the test was previously broken — `fn` should not see the outer `arguments`, because it was not an arrow function in input.js